### PR TITLE
jobs: Adds set -e and set -x flags to the renew_certificate job

### PR DESCRIPTION
**Pull request by claudiubelu**
``set -e`` tells the script to exit the script if any command returns a non-zero exit status.

``set -x`` is useful for debugging, essentially allowing us to trace the script execution.

### CODE REVIEW

**Comment by claudiubelu** [comment]
@maoo this PR needs approval before we can merge.

**Review by maoo — APPROVED** [review]

**Pull request merged**
